ics-frontend/src/app/header/header.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-toolbar color="primary">
+    <mat-toolbar-row>
+        <a mat-button routerLink="/image-classification">Tag your image</a>
+        <span class="spacer"></span>
```

Comment:
It's not a good practice to use such `spacer` elements especially when you can achieve the same effect just by using flexbox.

For example if add the following style declaration
```css
.mat-toolbar-row {
 display: flex;
 justify-content: space-between;
}
```
You can completely remove this `spacer` span (and its styles)

Reply:
adjusted